check_api/src/main/java/com/google/errorprone/matchers/Description.java

```diff
@@ -29,7 +29,11 @@
 import com.sun.source.tree.Tree;
 import com.sun.tools.javac.tree.JCTree;
 import com.sun.tools.javac.util.JCDiagnostic.DiagnosticPosition;
+
```

Comment:
nit: accidental newline? Or do we just let them come in?

Reply:
hmm, not sure where that came from, I can remove